src/System.Runtime/src/ApiCompatBaseline.uapaot.txt

```diff
@@ -2,3 +2,6 @@ Compat issues with assembly System.Runtime:
 TypeCannotChangeClassification : Type 'System.Decimal' is marked as readonly in the contract so it must also be marked readonly in the implementation.
```

Comment:
Nit: You can delete the existing content while you are on it. It should not be necessary anymore.